com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidBundler.java

```diff
@@ -682,13 +682,21 @@ private static File createAppBundleBaseZip(Project project, File outDir, File ap
     private static File createBundle(Project project, File outDir, File baseZip, ICanceled canceled) throws CompileExceptionError {
         logger.info("Creating Android Application Bundle");
         try {
+            BobProjectProperties projectProperties = project.getProjectProperties();
+            Boolean extractNativeLibs = projectProperties.getBooleanValue("android", "extract_native_libs");
+
             File bundletool = new File(Bob.getLibExecPath("bundletool-all.jar"));
             File baseAab = new File(outDir, getBinaryNameFromProject(project) + ".aab");
 
             File aabDir = new File(outDir, "aab");
             File baseConfig = new File(aabDir, "BundleConfig.json");
             try (BufferedWriter writer = new BufferedWriter(new FileWriter(baseConfig))) {
-                writer.write("{\"compression\":{\"uncompressedGlob\": [\"assets/game.arcd\"]}}");
+                String uncompressedGlob = "\"assets/game.arcd\"";
+                if (!extractNativeLibs) {
+                    uncompressedGlob += ",";
+                    uncompressedGlob += "\"lib/**/*.so\"";
+                }
+                writer.write("{\"compression\":{\"uncompressedGlob\": [" + uncompressedGlob + "]}}");
```

Comment:
At some point it will be useful to give access to that file as well as we do for manifest files, but it's good for now.